nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessPropagation.java

```diff
@@ -208,7 +208,7 @@ private NullnessStore lambdaInitialStore(
     NullnessStore environmentMapping =
         Objects.requireNonNull(
             environmentNullness.getEnvironmentMapping(underlyingAST.getLambdaTree()),
-            "no environment stored for lambda " + underlyingAST.getLambdaTree());
+            "no environment stored for lambda");
```

Comment:
separate, tiny optimization: I saw the string conversion of the lambda tree showing up in profiles